docs/adapters/http.md

```diff
@@ -40,14 +40,15 @@ Parameter         |             Description          | Required?
 `-format`         | HTTP data format, supports: `csv`, `json` and `jsonl` | No; defaults to HTTP response header `Content-Type`
 `-followLinkHeader` |  When set to true the [Web Linking](https://tools.ietf.org/html/rfc5988) feature is enabled which allows for automatic handling of paging as specified by the [RFC5988](https://tools.ietf.org/html/rfc5988). | No; default `false`
 `-pageParam`      | The name of the query parameter used to enable paging with multiple HTTP requests sent to the URL provided until response contains 0 records | No; defaults to paging behavior disabled
-`-pageStart`      | Initial value of the paging offset to set the `pageParam` to in the URL used to make the HTTP request | No; default: `0`
-`-pageUnit <request|record>` | If set to `request`, then each successive HTTP paging request increments the paging query parameter by one. If set to `record`, then each successive request increments the paging query parameter by the number of records returned in the previous request. | No; default: `request`
+`-pageStart`      | Initial value of the paging offset to set the `pageParam` to in the URL used to make the HTTP request. This is ignored when `-pageField` is set. | No; default: `0`
```

Comment:
I think it would be better to disallow pageStart if pageField is set just like we do for pageUnit.

Also FWIW I don't think we need to spell out that pageStart / pageUnit can't be used with pageField in the usage of all of them. How about we leave pageStart and pageUnit as they are and just have this description for pageField:

"Indicates the field name in the response that contains the offset for the next page. Cannot be used with pageStart or pageUnit."